src/foam/u2/borders/DetachedBorder.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+ foam.CLASS({
+  package: 'foam.u2.borders',
+  name: 'DetachedBorder',
```

Comment:
Why is it called 'Detached'?

Reply:
Because it is not linked to the trigger that controls its visibility.

Reply:
How is that evident from the code? Or is that just how it's used?

Reply:
Both really. In SectionedDetailPropertyView, the icon to show/hide it is not in the same div/parent unlike other collapsable borders. There is also no way to show/hide this view from within itself, which is something we do for other collapsing divs

Reply:
In FOAM, "Detach" has a specific meaning, which makes this confusing. Can you think of a different name? Maybe ExpandableBorder?

Reply:
Works, will update